configure

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+function usage
+{
+    echo "usage: ./configure [-DINSTALLDIR_LEMONADE=/path/to/install/LeMonADE/] [-DBUILDDIR=/path/to/build/LeMonADE/]"
+    echo "default build directory is ./build"
+    echo "default install directory is /usr/local"
+}
+
+#default values for build directory and install prefix
+PWD_PATH=$(pwd)
+BUILDDIR="build"
+
+#parse command line arguments
+for arg in "$@"; do
+    case $arg in
+        -h | --help)
+            usage         
+            exit
+            ;;
+            
+	-DINSTALLDIR_LEMONADE=*)	       	
+			INSTALLDIR_ARGUMENT=${arg}
+			INSTALLDIR=${arg#*=}
+			echo "Install directory set to "$INSTALLDIR
+			;;
+				
+	-DBUILDDIR=*)           
+			BUILDDIR=${arg#*=}
+			echo "Build directory set to "$BUILDDIR
+			;;
+
+	* )                     		
+			echo "unknown parameter"
+			usage         
+			exit
+
+
+    esac
+    shift           # shift the found arg away.
+done
+
+#create build directory if it does not exist
+if [ ! -d $BUILDDIR ]; then
+    mkdir -p $BUILDDIR
+fi
+
+
+(cd $BUILDDIR >/dev/null 2>&1 && cmake $INSTALLDIR_ARGUMENT $PWD_PATH)
+
+########### set up the wrapper makefile ##################
+
+cat > ./Makefile <<EOF
+
+##############################################################
+#               CMake Project Wrapper Makefile               #
+############################################################## 
+
+SHELL := /bin/bash
+RM    := rm -r
+
+all: ./${BUILDDIR}/Makefile
+	@ \$(MAKE) -C ${BUILDDIR}
+
+./${BUILDDIR}/Makefile:
+	@ (cd ${BUILDDIR} >/dev/null 2>&1 && cmake ${INSTALLDIR_ARGUMENT} ${PWD_PATH})
+
+clean:
+	@- (cd ${BUILDDIR} >/dev/null 2>&1 && cmake .. >/dev/null 2>&1)
```

Comment:
replace: @- (cd ${BUILDDIR} >/dev/null 2>&1 && cmake .. >/dev/null 2>&1)
to: @- (cd ${BUILDDIR} >/dev/null 2>&1)